lwallet/Main.hs

```diff
@@ -166,6 +167,19 @@ runCmd sendActions (Send idx outputs) CmdCtx{na} = do
     case etx of
         Left err -> putText $ sformat ("Error: "%stext) err
         Right tx -> putText $ sformat ("Submitted transaction: "%txaF) tx
+runCmd sendActions (SendDistr idx output distr) CmdCtx{na} = do
+    skeys <- getSecretKeys
+    etx <-
+        withSafeSigner (skeys !! idx) (pure emptyPassphrase) $ \mss ->
+        runEitherT $ do
+            ss <- mss `whenNothing` throwError "Invalid passphrase"
+            lift $ submitTxNoOverride
+                (immediateConcurrentConversations sendActions na)
+                ss
+                (one $ TxOutAux output distr)
+    case etx of
+        Left err -> putText $ sformat ("Error: "%stext) err
+        Right tx -> putText $ sformat ("Submitted transaction: "%txaF) tx
```

Comment:
http://i.imgur.com/ApNwUtZ.png
I don't find such copy-paste acceptable. Also it would be good to get rid of `EitherT`, because we use `ExceptT`.